### PR TITLE
Fix indentation in README

### PR DESCRIPTION
**Pull request by rickwierenga**

### CODE REVIEW

**Comment by WilliamHYZhang** [comment]
I'm just speculating here, but maybe the indent was because those lines are supposed to be the output of ```print(info)```, and not the actual code itself.

**Comment by rickwierenga** [comment]
@WilliamHYZhang you are right, my fault. I split the code blocks for better readability in https://github.com/tensorflow/datasets/pull/1319/commits/3707519f1c123654163bcb9df30e31c891273746.

File: README.md [hunk 1]
```diff
@@ -92,32 +92,34 @@ ds = mnist_builder.as_dataset(split='train')
 # dataset and its features
 info = mnist_builder.info
 print(info)
+```
```
Comment (WilliamHYZhang) [inline README.md]: Maybe add a line here, "output:" or "result:" to better distinguish?

**Comment by Conchylicultor** [comment]
Thanks for your contribution. Merging now.

**Pull request merged**